operator/README.md

```diff
@@ -18,11 +18,11 @@ architecture and a code overview, see [ARCHITECTURE.md](./ARCHITECTURE.md).
 The operator uses the [IstioOperator API](https://github.com/istio/api/blob/master/operator/v1alpha1/operator.proto), which has
 three main components:
 
-- [MeshConfig](https://github.com/istio/api/blob/master/mesh/v1alpha1/operator.proto) for runtime config consumed directly by Istio
+- [MeshConfig](https://github.com/istio/api/blob/baab1f25e9f7774ea784e2a919a3954c9d4b81c0/mesh/v1alpha1/config.pb.go#L215) for runtime config consumed directly by Istio
```

Comment:
Why do we use this specific commit id for README.md? 
You are right, the old link is invalid. But we'd better not use a fixed commit id for the README in master.

Reply:
I saw this in other places and felt a permanent link made more sense. Happy to update this though. Is it ok to use a line number?

Reply:
I think pointing to `master` and just the file makes the most sense; otherwise we'll soon have to change this again, when the line number reference breaks